.npmignore

```diff
@@ -1,7 +0,0 @@
-.idea
```

Comment:
Without this file, npm won't publish the bundle

Reply:
Done